packages/api-gateway-service/service.ts

```diff
@@ -16,6 +16,8 @@ if ( process.env.NODE_ENV === 'test' ) {
   dotenv.config( { path: './src/__tests__/.test.env' } );
 }
 
+console.log( process.env );
```

Comment:
This needs to be deleted

Reply:
Had added that for debugging on the qa deployment.

Removed it.